data/src/main/java/camp/nextstep/edu/github/data/service/GithubService.kt

```diff
@@ -0,0 +1,9 @@
+package camp.nextstep.edu.github.data.service
+
+import camp.nextstep.edu.github.data.dto.GithubRepositoryDto
+import retrofit2.http.GET
+
+interface GithubService {
```

Comment:
```suggestion
internal interface GithubService {
```
GithubService는 data 모듈에서만 사용해요.
그렇다면 internal 접근자를 붙여보면 어떨까요?